versioned_docs/version-4.0.0/tributech_agent/setup.mdx

```diff
@@ -0,0 +1,160 @@
+
+
+
+:warning: ***enrollment.key*** file in the referenced folder must contain the `enrollment key` in the format:
+```
+-----BEGIN PRIVATE KEY-----
+MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDB910tiUPf3jp3
+...
+JoTeVZlyPgWSr6ckDiYrXZCINfeupxIpUNa2dOTssJ7frddsOc65TjYaEGtQFhN9
+UuhRTNXXB3LPeUekmFqAFw==
+-----END PRIVATE KEY-----
+```
+
+:warning: ***enrollment.crt*** file in the referenced folder must contain the `enrollment certificate` in the format:
+
+```
+-----BEGIN CERTIFICATE-----
+MIIEhDCCAmygAwIBAgIVAMJHgTEKo6BAjM2x17MzqEoBAAAAMA0GCSqGSIb3DQEB
+...
+rTGtd7jookqEziPG7j9oN6Q4jQZI/fGeTDpz5JvEPriFWg4niwdZZVTVi4Axk0wT
+Xb7SLA93BKc=
+-----END CERTIFICATE-----
+```
+
+
+### Docker volumes
+Docker Volumes are the preferred mechanism for persisting data generated by and used by Docker containers.
+To make local certificates folders available to a docker container we need to add the mount to our `tributech-agent` service.
+The folder for the certificates can be adjusted as needed (`./enrollment`) only the `:/app/enrollment` needs to
+stay the same. 
+
+<CodeBlock className="language-yml" title="docker-compose.yml section">{DockerComposeVolumesExample}</CodeBlock>
+
+
+### Environment Variables
+The second way to add enrollment certificates to a Tribteuch Agent is to use [environment variables](https://docs.docker.com/compose/environment-variables/set-environment-variables/). The content of the certificate files needs to be converterted into a base64 encoded string without whitespaces, i.e. detailed information on how to convert a file into a base64 representation can be found here for [Windows](https://github.com/tributech-solutions/tributech-enrollment/tree/main/Windows) and [Unix](https://github.com/tributech-solutions/tributech-enrollment/tree/main/Linux). We then add the base64 encoded content of the `enrollment.crt` (Certificate) as `EnrollmentOptions__EnrollmentCertBase64` value and the `EnrollmentOptions__EnrollmentKeyBase64` contains base64 encoded `enrollment.key` content.
+
+<CodeBlock className="language-yml" title="docker-compose.yml section">{DockerComposeEnviromentExample}</CodeBlock>
+
+## Starting a Tributech Agent
+After we have prepared the Tributech Agent certificates for the enrollment we can now startup whole environment.
+We will use in our example the certifcates reference via environment variables. If docker volumes are preferred
+the `tributech-agent-simulated` in the comming examples need to be adjust like described in the previous section.
+
+:warning: We will use in our examples for the setup documentation different Container Images and configurations compared to the [Quickstart](quickstart.mdx). Please use a new empty folder for the files to prevent any configuration mix.
```

Comment:
```suggestion
:warning: In our examples for the setup documentation, we will use different container images and configurations compared to the [Quickstart](quickstart.mdx). Please use a new empty folder for the files to prevent any configuration mix.
```